src/renderers/server-render.jsx

```diff
@@ -190,19 +182,15 @@ const makeRenderer = (
 	})
 	.first(state => state.preRenderChecklist.every(isReady => isReady));  // take the first ready state
 
-	console.log(JSON.stringify({
-		message: `Dispatching RENDER for ${request.url.href}`,
-		type: 'dispatch',
-		info: {
-			url: request.url,
-			method: request.method,
-			id: request.id,
-		}
-	}));
-	store.dispatch({
+	const action = {
 		type: SERVER_RENDER,
 		payload: url,
-	});
+	};
+	server.app.logger.debug(
```

Comment:
What's the advantage of using `server.app.logger` instead of just the logger module directly? Is it just one fewer import? The reason I ask is because I think it will be unclear for others working in this code the difference (none) and/or when server.app is available and when it's not.